flow-server/src/main/resources/com/vaadin/flow/server/frontend/Flow.tsx

```diff
@@ -418,3 +419,4 @@ export const createWebComponent = (tag: string, props?: Properties, onload?: ()
     return React.createElement(tag);
 };
 
+//%buildRouteFunction%
```

Comment:
no new line.
Just for usability I would perhaps have here 

```suggestion
export const buildRoute = (views: any): any => {
  //%buildRouteFunction%
}
```
 and the replace would just be `return views;` or `return toReactRouter(views);` to have as little coding in strings as possible. 
 
 Didn't see a need for `// @ts-ignore`